data_structures/Cargo.toml

```
@@ -10,20 +10,21 @@ workspace = ".."
 bech32 = "0.7.1"
 byteorder = "1.3.2"
 cbor-codec = "0.7.1"
+chrono = "0.4.9"
 failure = "0.1.6"
 lazy_static = "1.4.0"
+log = "0.4.8"
 num_enum = "0.4.1"
 protobuf = { version = "2.8.1", features = ["with-serde"] }
 protobuf-convert = "0.1.1"
 rand = "0.7.2"
 serde_cbor = "0.10.2"
-toml = "0.5.3"
-log = "0.4.8"
 secp256k1 = "0.15.5"
+toml = "0.5.3"
 vrf = "0.2.2"
+
 witnet_reputation = { path = "../reputation", features = ["serde"] }
 witnet_protected = { path = "../protected" }
-chrono = "0.4.9"
 
 [dependencies.partial_struct]
```

While we are at it, could you also remove all the usages of this syntax?

```suggestion
partial_struct = { path = "../partial_struct" }
```